spaceship/lib/spaceship/connect_api/models/app.rb

```diff
@@ -236,7 +238,10 @@ def get_in_review_app_store_version(platform: nil, includes: Spaceship::ConnectA
       def get_pending_release_app_store_version(platform: nil, includes: Spaceship::ConnectAPI::AppStoreVersion::ESSENTIAL_INCLUDES)
         platform ||= Spaceship::ConnectAPI::Platform::IOS
         filter = {
-          appStoreState: Spaceship::ConnectAPI::AppStoreVersion::AppStoreState::PENDING_DEVELOPER_RELEASE,
+          appStoreState: [
+            Spaceship::ConnectAPI::AppStoreVersion::AppStoreState::PENDING_APPLE_RELEASE,
+            Spaceship::ConnectAPI::AppStoreVersion::AppStoreState::PENDING_DEVELOPER_RELEASE
+          ].join(','),
```

Comment:
Actually I don't know enough to judge this part, but that's lack of rails knowledge probably 🙈 I will trust that this will just work 😉